### PR TITLE
Fixing Swarm Install with Machine doc

### LINKED ISSUE #1767: Docker swarm doesn't work with latest 0.6.0 docker-machine.
After following the tutorial at https://docs.docker.com/swarm/install-w-machine/
here's the error,
$ eval $(docker-machine env swarm)
Host does not exist: "swarm"

Referred to bug https://github.com/docker/swarm/issues/1664 when i found my docker-machine version to be 0.5.6 and upgraded it. Still didn't solve the issue with latest 0.6.0

### PR DESCRIPTION
**Pull request by nishanttotla**
This fixes #1767 and partially addresses #1771.

Changes include mounting the certs directory into Swarm manager container and updating port numbers for `swarm join`.

Signed-off-by: Nishant Totla <nishanttotla@gmail.com>

### CODE REVIEW

**Comment by nishanttotla** [comment]
cc @moxiegirl @dongluochen @vieux

File: docs/install-w-machine.md [hunk 1]
```diff
@@ -101,20 +101,20 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
...
 
 2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container as the primary Swarm manager on `manager`.
 
-        $ docker run -d -p <your_selected_port>:2375 -t swarm manage token://<cluster_id> >
+        $ docker run -d -p <your_selected_port>:2375 -t -v /var/lib/boot2docker:/certs:ro swarm manage --tlsverify --tlscacert=/certs/ca.pem --tlscert=/certs/server.pem --tlskey=/certs/server-key.pm token://<cluster_id>
 
     For example:
 
-        $ docker run -d -p 2375:2375 -t swarm manage token://0ac50ef75c9739f5bfeeaf00503d4e6e
+        $ docker run -d -p 3375:2375 -t -v /var/lib/boot2docker:/certs:ro swarm manage --tlsverify --tlscacert=/certs/ca.pem --tlscert=/certs/server.pem --tlskey=/certs/server-key.pm swarm manage token://0ac50ef75c9739f5bfeeaf00503d4e6e
```
Comment (vieux) [inline docs/install-w-machine.md]: do you mind using `3376` since it's using TLS ?

File: docs/install-w-machine.md [hunk 2]
```diff
@@ -134,17 +134,21 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
...
-1. Connect the Docker Client to the Swarm.
+1. Connect the Docker Client to the Swarm by updating the `DOCKER_HOST` environment variable.
 
-        $ eval $(docker-machine env swarm)
+        $ DOCKER_HOST=<manager_ip>:<your_selected_port>
 
-    Because Docker Swarm uses the standard Docker API, you can connect to it using  Docker Client and other tools such as Docker Compose, Dokku, Jenkins, and Krane, among others.  
+    For the current example, the `manager` has IP address `192.168.99.100` and we selected port 3375 for the Swarm manager.
+
+        $ DOCKER_HOST=192.168.99.100:3375
```
Comment (vieux) [inline docs/install-w-machine.md]: same here

Comment (nishanttotla) [inline docs/install-w-machine.md]: fixed both, thanks.

**Comment by vieux** [comment]
LGTM

File: docs/install-w-machine.md [hunk 3]
```diff
@@ -101,20 +101,20 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
...
-        manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1   
+        agent1    -        virtualbox   Running   tcp://192.168.99.102:2376           v1.9.1
+        agent2    -        virtualbox   Running   tcp://192.168.99.103:2376           v1.9.1
+        manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1
 
 
 2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container as the primary Swarm manager on `manager`.
 
-        $ docker run -d -p <your_selected_port>:2375 -t swarm manage token://<cluster_id> >
+        $ docker run -d -p <your_selected_port>:2375 -t -v /var/lib/boot2docker:/certs:ro swarm manage --tlsverify --tlscacert=/certs/ca.pem --tlscert=/certs/server.pem --tlskey=/certs/server-key.pm token://<cluster_id>
```
Comment (dongluochen) [inline docs/install-w-machine.md]: I'd suggest not using 2375 at all, like
`docker run -d -p <your_selected_port>:3376 -t -v /var/lib/boot2docker:/certs:ro swarm manage -H 0.0.0.0:3376 --tlsverify --tlscacert=/certs/ca.pem --tlscert=/certs/server.pem --tlskey=/certs/server-key.pm token://<cluster_id>`

Comment (nishanttotla) [inline docs/install-w-machine.md]: Fixed.

**Comment by dongluochen** [comment]
LGTM

**Comment by moxiegirl** [comment]
LGTM

**Comment by moxiegirl** [comment]
Thanks for these fixes @nishanttotla

**Pull request merged**